util/feeding.py

```diff
@@ -42,9 +43,12 @@ def samples_to_mfccs(samples, sample_rate):
 
 
 def audiofile_to_features(wav_filename):
+    global is_pertub
     samples = tf.io.read_file(wav_filename)
     decoded = contrib_audio.decode_wav(samples, desired_channels=1)
-    features, features_len = samples_to_mfccs(decoded.audio, decoded.sample_rate)
+    if is_pertub:
```

Comment:
if I read correctly, you're breaking the flow in case `is_perturb=False`, since we pass `samples` from `tf.io.read_file` to `samples_to_mfccs` instead of passing `decoded.audio`